Contributors.md

```diff
@@ -0,0 +1,76 @@
+
+## Prerequisites
+
+Before you start, make sure you have:
+
+- Read the [README.md](./README.md) (best quickstart guide for using the project) and can build, install, and run the tests yourself from those instructions (from both the main and development branch).  This is the assumed baseline for any issues, proposed features, or pull requests that you wish you contribute.
+- Are somewhat familiar with the [KERI, ACDC, CESR standards and the KERI whitepapers](https://github.com/WebOfTrust/keri).  Jargon isn't always the best, but it does speed communication with maintainers whose time is often in short supply and appreciate clarity in all things.  A very helpful website [kerisse.org](https://kerisse.org) has been set up to ease this process if you don't necessarily have the time to read all of those things. However, an in depth knowledge will surely help if you are looking to make a lot of contributions. 
+
+## How to Start Contributing
+The KERI community welcomes [all kinds of contributions](https://opensource.guide/how-to-contribute/).
+
+For simple contributions like fixing typos sometimes you can just submit a naked PR with a title like "I'm fixing a spelling issue in x,y,z".  If these PRs are small often times they'll just get merged without further review.  Typos or fixing names that were mistakenly applied are human nature.
+
+A really simple way to contribute is just to spread the word.  Let people know about KERI, create a welcoming community to newcomers by actively participating and helping with what you know, and making cools things and talking about them is helpful to grow.
+
+Issues and discussions are meant to be discussed, commented upon.  Even a +1 to an issue that you're also experiencing can help the core maintainers decide where to focus their energies.  Discussions can help provide clarity if things aren't exactly clear.  Feel free to contribute (but also note the homework in [PREREQUISITES](#prerequisites), its frustrating for people to explain things that are already explained elsewhere).
+
+However, this particular file is more about contributing code.  The general principal in this (and all open source really) is if you can think of a way that things might be better, instead of just suggesting it or discussing it, often times its a good idea to do some work to show what/how/when an idea might look like or contributing code/scripts/tests/issues.  Code >> discussion 9 times out of ten.  Balanced of course with the caveat not to go off into the ivory tower for months and do lots of work before presenting your work and finding out that the community doesn't like it.  Small prototypes/examples are often best.  The rest of the details of contributing code are below.
+
+## Contribution Guidelines
+
+### Code Style/Conventions
+[PythonStyleGuide](./ref/PythonStyleGuide.md)  
+
+### Commit Message Guidelines
+There are no hard and fast guidelines (TODO: is this true?) but it is helpful to:
```

Comment:
I mean we don't have rules, but a descriptive to the point message is ideal.

"Adds selective disclosure mechanism"
"Updates test coverage for coring"

We should discuss if we want to start requiring sign off.